src/pkg/build/builder_sif.go

```diff
@@ -112,8 +38,28 @@ func NewSifBuilderJSON(r io.Reader, imagePath string) (b *SIFBuilder2, err error
 		}
 	}
 
-	b.def = d
-	b.path = imagePath
+	return NewSIFBuilder(imagePath, d)
+
+	/*
+		b.Def = d
+		b.ImagePath = imagePath
+		b.tmpfs, err = image.TempSandbox(path.Base(imagePath) + "-")
+		if err != nil {
+			return b, err
+		}
+
+		uri := d.Header["bootstrap"] + "://" + d.Header["from"]
+		b.p, err = NewProvisionerFromURI(uri)
+
+		return b, err*/
```

Comment:
I'd like to see the commented-out code removed from this file. It clutters things up, and doesn't aid in understanding the code.

Reply:
Sure, just forgot to delete!